src/App.vue

```diff
@@ -378,28 +399,30 @@
     }
   }
 
-  a, .bookmarkUrlDisplay {
-    &.jumpboxLink::after{
-      @extend %fa-icon;
-      @extend .fas;
-      content: fa-content($fa-var-chevron-right);
-    }
-    &[href^="http://"], &[href^="https://"], &[href^="mailto"], &[href$=".pdf"], &[href$=".html"] {
-      &:not([href*="127.0.0.1"]):not([href*="localhost"]):not([href*="assets://"])::after{
+  .electron {
```

Comment:
Outlink notes aren't practical to do in CSS for a web app because it would need to know its own domain at compile time, so only do that in the desktop app.